app/models/reservation.rb

```diff
@@ -103,6 +103,17 @@ class Reservation < ActiveRecord::Base
       .select('reservations.*, equipment_models.category_id as category_id')
   }
 
+  # for status modifying jobs
+  scope :missed_not_emailed, ->() { missed.not_flagged(:missed_email_sent) }
+  scope :newly_missed, ->() { reserved.past_date(:start_date) }
+  scope :newly_overdue, ->() { not_overdue.past_date(:due_date) }
```

Comment:
This doesn't specify that it is checked out, correct? I believe that was part of the original query / task.